spec/std/http/request_spec.cr

```diff
@@ -336,6 +336,10 @@ module HTTP
         Request.new("GET", "//foo:bar").path.should eq "//foo:bar"
       end
 
+      it "parses with only leading with double slash" do
+        HTTP::Request.new("GET", "//").path.should eq "/"
```

Comment:
There are two possible expected scenarios for this case: `""` or `/`.  
In older Crystal versions, the expected value was `/`.